packages/api-docs/src/index.js

```diff
@@ -74,7 +74,7 @@ async function generateApiForFiles(files) {
 }
 
 export default async function (relDir, outputMd) {
-  const files = await glob(path.resolve(process.cwd(), relDir));
+  const files = await glob(path.resolve(process.cwd(), relDir.replace(/^'/, '').replace(/'$/, '')));
```

Comment:
this looks a bit mad so could do with a comment, or even better breaking in to a named function with a name that explains its purpose

Reply:
i guess the function could be called `dequote` - and can be simplified a little to be this:
```
function dequote(string) {
  return string.replace(/^'(.*)'$/, '$1');
}
```